### PR TITLE
Revert "PERF-5941 : cleanup perf test coverage"

### PR DESCRIPTION
**Pull request by thessem**
Reverting mongodb/genny#1267 as it breaks `mongo_perf_standalone_genny_InsertRemove
System` in the DSI merge queue. We need to remove this dependency first. See [here](https://spruce.mongodb.com/task/dsi_linux_microbenchmarks_standalone_arm.2023_09_mongo_perf_repl_genny_InsertRemove_patch_1e3e774acd62c0f5944cd0a3a2cacf6b2ffc7b46_670757c182b7b8000742bc4f_24_10_10_04_27_49/logs?execution=0) and [here](https://spruce.mongodb.com/task/dsi_linux_microbenchmarks_standalone_arm.2023_09_mongo_perf_standalone_genny_InsertRemove_patch_1e3e774acd62c0f5944cd0a3a2cacf6b2ffc7b46_670757c182b7b8000742bc4f_24_10_10_04_27_49/logs?execution=0) in a PR that is unrelated to these for examples.

### CODE REVIEW

File: evergreen/system_perf/7.3/genny_tasks.yml
Comment (PyGeek03) [inline evergreen/system_perf/7.3/genny_tasks.yml:1]: Just for safety, maybe we should not delete anything in this revert?

**Review by PyGeek03 — COMMENTED** [review]
Do we need to revert the whole of #1267? `mongo_perf_standalone_genny_InsertRemove` only needs the file "src/workloads/scale/InsertRemove.yml". Also, would be nice if there's a Jira ticket for fixing this.

**Comment by thessem** [comment]
> only needs the file "src/workloads/scale/InsertRemove.yml".

This reversion PR is an attempt at getting the DSI merge queue working again before my business day ends, a more finely tuned one is preferable if somebody is able to do it, this PR should be closed. A more finely tuned PR could be in the DSI repository.

> if there's a Jira ticket for fixing this.

The Jira ticket should be more about ownership or a load-bearing-comment of any files that DSI requires so this doesn't accidentally happen. I don't think people should need to run integration tests in another project/repository!

**Comment by PyGeek03** [comment]
> if there's a Jira ticket for fixing this.

By "this" I mean the breakage of DSI commit queue due to the removal of "src/workloads/scale/InsertRemove.yml" (for which I've filed [DEVPROD-11966](https://jira.mongodb.org/browse/DEVPROD-11966)), not a long-term fix. Sorry for the unclear language.

**Review by jimoleary — APPROVED** [review]
LGTM

**Review by PyGeek03 — APPROVED** [review]

**Comment by jawwadasghar** [comment]
@thessem apologize for any chaos I caused. 
**If the merge queue is fixed now, can we not revert this change?** 

We would like to leave out the remaining genny workloads that were removed and wait ~1 week to evaluate if we want to re-add them or not. 
cc @brad-devlugt @jimoleary

**Comment by thessem** [comment]
Closed in lieu of https://github.com/mongodb/genny/pull/1271

**Pull request closed without merging**